src/components/endowed-positions/EndowedPositionsSearch.js

```diff
@@ -0,0 +1,36 @@
+import React, { useCallback, useState } from 'react';
+import { navigate } from 'gatsby';
+import SbEditable from 'storyblok-react';
+
+const EndowedPositionsSearch = (props) => {
+  const [searchTerm, setSearchTerm] = useState;
```

Comment:
Currently, when you're on Storyblok, you'll probably see this error when you create your search component as a new entry
![Screen Shot 2022-08-23 at 4 24 12 PM](https://user-images.githubusercontent.com/42749717/186286459-f2327286-f4be-4931-b46d-19abc8d1dc84.png)

This can be fixed by using the correct syntax for useState
```suggestion
  const [searchTerm, setSearchTerm] = useState('');
```